background_scripts/main.coffee

```diff
@@ -5,6 +5,7 @@ root = exports ? window
 chrome.runtime.onInstalled.addListener ({ reason }) ->
   # See https://developer.chrome.com/extensions/runtime#event-onInstalled
   return if reason in [ "chrome_update", "shared_module_update" ]
+  return if Utils.isFirefox()
```

Comment:
Why to do nothing on `runtime.onInstalled` if the browser is Firefox?

Reply:
Oh, it does fail. Thanks! Then I want to test the update notification if you haven't, in order to ensure nothing is working on Firefox.

Reply:
> test the update notification if you haven't

I haven't. Go for it.